app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -7,12 +7,18 @@ window.GOVUK.analyticsGA4.analyticsModules = window.GOVUK.analyticsGA4.analytics
   'use strict'
 
   var Ga4LinkTracker = {
-    init: function () {
+    init: function (config = {}) {
```

Comment:
I think having named function parameters like this is an ES6 thing? If so it's not supported by some older IEs that we want to include. Can this be rewritten somehow? (assuming I'm correct, if not just ignore me).

Reply:
Yep, you're correct - will be simple to fix :+1: